go/ql/src/change-notes/2025-03-20-logging-false-positive-type-format-specifier.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* False positives in "Log entries created from user input" (`go/log-injection`) and "Clear-text logging of sensitive information" (`go/clear-text-logging`) which correspond to the verb `%T` in a format specifier have been removed. There may also be more results in "Use of constant `state` value in OAuth 2.0 URL" (`go/constant-oauth2-state`).
```

Comment:
Should this be less results, for the same reason?

Reply:
In `go/constant-oauth2-state`, arguments to loggercalls are used to rule out some results. To less arguments to loggercalls means more results, potentially. (In practice I doubt there will be any alert changes for that query.)